src/utils.ts

```diff
@@ -168,7 +168,8 @@ export async function getFilesFromSourceFile({
   return lines
 }
 
-export function tempfile(extension = ''): string {
+export function tempfile(extension = ''): Promise<string> {
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/promise-function-async](https://typescript-eslint.io/rules/promise-function-async)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Functions that return promises must be async.
```suggestion
export async function tempfile(extension = ''): Promise<string> {
```